data/categories.txt

```diff
@@ -10091,7 +10091,7 @@ nl:4Outdooruitrusting|trekking|klimmen|kamperen
 fi:Ulkoiluvarusteet|vaellus|kiipeily|retkeily
 fr:Matériel de loisirs de plein air|Randonnée|Escalade|Camping
 de:4Outdoor-Ausrüstung|Trekking|Klettern|Camping
-hu:Túrafelszerelés|trekking|mászás|kemping|üzlet
+hu:Túrafelszerelés|trekking|mászás|kemping|üzlet|Kültéri felszerelés
```

Comment:
`Kültéri felszerelés` is in the wrong context here, `kültéri felszerelés` would be like a lamp that you put on the outside wall of your house, but that does not mean that you can use those equipments for [outdoor activities](https://en.wikipedia.org/wiki/Outdoor_recreation).
If you must insert one more word here, then perhaps `Szabadtéri felszerelés` is a much better fit.

Reply:
Same as above. What is the best translation for [type.shop.outdoor] ?

Reply:
I would go with `Túrafelszerelés`.